cms-plugin/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/attribute/action/link/LinkAttributeAction.java

```diff
@@ -224,10 +224,10 @@ public void setElementIndex(int elementIndex) {
 	}
 	
 	@Override
-	public String getLangCode() {
+	public String getLinkLangCode() {
```

Comment:
I think that the new method name doesn't match his behavior, maybe is `getLangCodeOfLink`  a better name? ... I mean it returns the lang code .... not the link

Reply:
Fixed